src/main/java/seedu/address/ui/UiManager.java

```diff
@@ -20,7 +24,7 @@
     public static final String ALERT_DIALOG_PANE_FIELD_ID = "alertDialogPane";
 
     private static final Logger logger = LogsCenter.getLogger(UiManager.class);
-    private static final String ICON_APPLICATION = "/images/address_book_32.png";
+    private static final String ICON_APPLICATION = "/images/icon.png";
```

Comment:
colab_icon_32 would be easier to differentiate.  32 being an example size.

Reply:
agreed, will check the other images and relabel them accordingly too.